pyproject.toml

```diff
@@ -30,10 +30,11 @@ itsdangerous = "^2.1.2"
 starlette-oauth2-api = "^0.2.6"
 httpx = "^0.20.0"
 
-fasttext = "^0.9.2"
+# fasttext = "^0.9.2"
```

Comment:
commenting this out for now, for some reason I am installing fasttext is causing errors and I will try to look into this.